Makefile

```diff
@@ -104,6 +104,12 @@ ifeq ("$(TARGETOS)", "trusty")
 	TARGETGOARCH := $(HOSTARCH)
 endif
 
+ZLIBFLAGS :=
+
+ifeq ("$(TARGETOS)", "linux")
```

Comment:
This should be in sys/targets/targets.go.
Then it will also affect reproducers, and how tests build executor.

Reply:
Thanks for this! I notice [here](https://github.com/google/syzkaller/blob/5c716ff/sys/targets/targets.go#L563) that we may target distributions which do not support static libraries. If I understand correctly, this may cause an issue for our proposed `zlib` solution? I'm not sure of how best to include it only in those builds which will not produce missing header/linker errors.